v03_pipeline/var/test/reference_data/test_combined_mito_2.ht/README.txt

```diff
@@ -0,0 +1,3 @@
+This folder comprises a Hail (www.hail.is) native Table or MatrixTable.
+  Written with version 0.2.130-bea04d9c79b5
+  Created at 2024/11/06 12:09:16
```

Comment:
I made a copy of the original test_combined_mito_1 table (has all mito annotations) and updated the path, version, and added apogee2_score